### PR TITLE
[15.0][ADD] web_ir_actions_multi_print: New module

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa  TT51076

This module allows print more than one report at the same time. Based on stock print auto delivery in v17

ping @chienandalu

### CODE REVIEW

File: web_ir_actions_multi_print/sourceType: module
Comment (chienandalu) [inline web_ir_actions_multi_print/sourceType: module:1]: what's this file? :thinking:

File: web_ir_actions_multi_print/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+This module provides a way to trigger more than print report render download at the
+same time.
+This module is based on Odoo 17 delivery report automation
```
Comment (chienandalu) [inline web_ir_actions_multi_print/readme/DESCRIPTION.rst]:
```suggestion
This is a backport of the same functionality existing in Odoo 17.
```

File: web_ir_actions_multi_print/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2024 Tecnativa - Sergio Teruel
+# License AGPLv3.0 or later (https://www.gnu.org/licenses/agpl-3.0.en.html).
+
+{
+    "name": "Web Actions Multi Print",
+    "summary": "Enables multi print actions",
+    "category": "Web",
+    "version": "15.0.1.0.0",
+    "license": "LGPL-3",
+    "author": "Tecnativa, Odoo Community Association (OCA)",
```
Comment (chienandalu) [inline web_ir_actions_multi_print/__manifest__.py]:
```suggestion
    "author": "Odoo S.A., Tecnativa, Odoo Community Association (OCA)",
```

File: web_ir_actions_multi_print/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2024 Tecnativa - Sergio Teruel
```
Comment (chienandalu) [inline web_ir_actions_multi_print/__manifest__.py:1]:
```suggestion
# Copyright 2024 Tecnativa - Sergio Teruel
# Copyright 2024 Odoo S.A.
```

**Comment by sergio-teruel** [comment]
Changes done!!

File: web_ir_actions_multi_print/sourceType: module
Comment (sergio-teruel) [reply web_ir_actions_multi_print/sourceType: module:1]: pre-commit require it when you use import... is autogenerated

Comment (pedrobaeza) [reply web_ir_actions_multi_print/sourceType: module:1]: I don't think so... it seems some misconfiguration in your scaffolding (or a bug in DCT).

**Comment by sergio-teruel** [comment]
Rebase done to get lastest copier template.

**Review by chienandalu — APPROVED** [review]

**Review by pedrobaeza — COMMENTED** [review]
Seeing the code, isn't this usable for any possible action (print, client action, server action...)?

And why not using https://github.com/OCA/web/tree/15.0/web_ir_actions_act_multi ?

**Comment by sergio-teruel** [comment]
Sorry, after testing it and try to run this modulo is not wortking...

**Comment by sergio-teruel** [comment]
Close this PR. we can use directly https://github.com/OCA/web/tree/15.0/web_ir_actions_act_multi

**Pull request closed without merging**